extensions/wikia/MercuryApi/MercuryApiController.class.php

```diff
@@ -397,34 +393,52 @@ public function getArticleFromMarkup() {
 		$this->response->setVal( 'wikiVariables', $wikiVariables );
 	}
 
-	/**
-	 * @throws NotFoundApiException
-	 * @throws BadRequestApiException
-	 */
-	public function getArticle() {
+	public function getPage() {
```

Comment:
this method has almost 100 lines, I think there are some places here and there we could move to a separate, maybe reusable ^^ methods?